public/documentation/getting_started/remote_debugging.md

```diff
@@ -1,6 +1,6 @@
 # Remote debugging
 
-Web applications running under Crosswalk can be debugged remotely using the [Chrome dev tools](https://developer.chrome.com/devtools/index). 
+Web applications running under Crosswalk can be debugged remotely using [Chrome dev tools](https://developer.chrome.com/devtools/index). 
 
```

Comment:
Why remove "the" ?  I know it could be written either way but "the" flows well in this sentence.  Google aslo uses "the" before referencing their tools.  https://developer.chrome.com/devtools